src/drivers/pwm_out_rc_in/pwm_out_rc_in.cpp

```diff
@@ -46,8 +46,8 @@
 
 #include <drivers/drv_hrt.h>
 #include <drivers/drv_mixer.h>
-#include <systemlib/mixer/mixer.h>
-#include <systemlib/mixer/mixer_multirotor.generated.h>
+#include <lib/mixer/mixer.h>
+#include <lib/mixer/mixer_multirotor.generated.h>
```

Comment:
Should we change this include to use `mixer_multirotor_normalized.generated.h` to make sure the same header is used everywhere?

Reply:
Good catch, I fixed it also in  
`src/drivers/pwm_out_rc_in/pwm_out_rc_in.cpp`, 
`src/drivers/snapdragon_pwm_out/snapdragon_pwm_out.cpp`, and 
`src/platforms/qurt/fc_addon/uart_esc/uart_esc_main.cpp`. 

`mixer_multirotor_normalized.generated.h` is implementation detail, what is the reason for including it?

Reply:
I think to create the default mixer: https://github.com/jlecoeur/Firmware/blob/560f229eb6be6db73ea790e7607fbfda2148b041/src/drivers/pwm_out_rc_in/pwm_out_rc_in.cpp#L180
I agree, there probably should be a better way to do that.